docs/userGuide/reusingContents.md

```diff
@@ -59,14 +59,12 @@ If you are using Git for version control, you can set up the sub-site repository
 </span>
 </include>
 
+#### Excluding Files from Rendering as Pages
 
-<hr><!-- ======================================================================================================= -->
-
-## Creating Custom Fragments
-
-**MarkBind supports creating custom fragments**. A fragment is a piece of content that can be reused across multiple pages. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but rather than reusing content from a _sub-site_, content is reused from fragments which can be excluded from page generation.
+**MarkBind supports the exclusion of files from rendering as pages**. This allows for the ease of creation of <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip>. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but instead of a _sub-site_, content is reused from fragments which are excluded from page generation. This facilitates using repeated content across pages.
```

Comment:
```suggestion
**MarkBind supports the exclusion of files from rendering as pages**. For example, you can exclude files containing <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip> of reusable content from page generation.
```

Reply:
I think there's a typo in the original issue, which says that this part should:
> Move it ~~under~~ above the "Reusing Contents Across Sites" section, under "Includes", as it is also useful with just includes, even without subsites.

Reply:
Oh I see, I think this makes more sense as well! I have made the relevant changes in the latest commit!